src/engagement_db_to_analysis/traced_data_filters.py

```diff
@@ -65,45 +65,41 @@ def rqa_time_range_filter(user, messages_traced_data, pipeline_config):
     return filtered
 
 
-def filter_test_participants(user, participants_traced_data_map, test_participant_uuids):
+def filter_test_participants_messages(user, messages_traced_data, test_participant_uuids):
```

Comment:
I think this can just be filter_test_messages(

Reply:
Yup, updated thanks.